wikimedia/wikipedia-ios
# Talk pages - scroll to comment after posting

**Phabricator:** 
N/A

### Notes
This PR scrolls to a particular comment view after a user successfully posts a new reply. This is based off of #4331, so please do not merge until that one is merged.

### Test Steps
1. Post a reply on a topic. Try the blue Reply button when it's near the bottom of the scroll view - it should append a reply off screen.
2. Confirm that after a brief delay after posting, the new comment scrolls into view.
